docs/src/pages/components/slider-styled/InputSlider.js

```diff
@@ -1,22 +1,20 @@
 import * as React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
 import Grid from '@material-ui/core/Grid';
 import Typography from '@material-ui/core/Typography';
 import Slider from '@material-ui/lab/SliderStyled';
-import Input from '@material-ui/core/Input';
+import MuiInput from '@material-ui/core/Input';
 import VolumeUp from '@material-ui/icons/VolumeUp';
 
-const useStyles = makeStyles({
-  root: {
-    width: 250,
-  },
-  input: {
-    width: 42,
-  },
-});
+const Root = styled('div')`
+  width: 300px;
+`;
```

Comment:
Should we use the JavaScript API instead?

```suggestion
const Root = styled('div')({
  width: 300,
});
```